### PR TITLE
Remove mentions of the upgrade time from the warnings

### PR DESCRIPTION
**Pull request by paulomarg**
### WHY are these changes introduced?

Since we can't say for sure how long the update process will actually take for the user, we should make no mention of it in the warning messages.

### WHAT is this pull request doing?

Simply removing the unneeded text from the messages.

### CODE REVIEW

**Comment by paulomarg** [comment]
- 1 / 2 - How about: 

        {{x}} The Shopify App CLI was migrated to the Shopify CLI. You’ll need to migrate your installation to continue using it.
            Please visit this page for complete instructions:
            {{underline:https://shopify.github.io/shopify-app-cli/migrate/}}

- 3 - I'm not sure we can rename the repository without breaking all of the 'old' installs because it pulls the code straight from github - any thoughts @mkevinosullivan ?
- 4 - I'll fix that as per above

**Comment by paulomarg** [comment]
Scratch comment 3, I think we may be able to rename it after all without breaking clones, though it's not a super clean process: https://help.github.com/en/github/administering-a-repository/renaming-a-repository . Considering people would get the migration message if they updated, we might be able to pull it off.

File: lib/shopify-cli/messages/messages.rb [hunk 1]
```diff
@@ -267,7 +267,7 @@ module Messages
           DEVELOPMENT
 
           shell_shim: <<~MESSAGE,
-          {{x}} This version of Shopify App CLI is no longer supported. You’ll need to upgrade to continue using it. This process typically takes a few minutes.
+          {{x}} This version of Shopify App CLI is no longer supported. You’ll need to upgrade to continue using it.
```
Comment (gfscott) [inline lib/shopify-cli/messages/messages.rb:270]:
```suggestion
          {{x}} This version of Shopify App CLI is no longer supported. You’ll need to migrate to the new CLI version to continue.
```

File: lib/shopify-cli/messages/messages.rb [hunk 2]
```diff
@@ -267,7 +267,7 @@ module Messages
...
 
             Please visit this page for complete instructions:
             {{underline:https://shopify.github.io/shopify-app-cli/upgrade/}}
```
Comment (gfscott) [inline lib/shopify-cli/messages/messages.rb:273]:
```suggestion
            {{underline:https://shopify.github.io/shopify-app-cli/migrate/}}
```

@paulomarg I'll leave it to you to make the change the repo handle, if that's what we end up doing...

Comment (paulomarg) [reply lib/shopify-cli/messages/messages.rb]: Yeah that sounds good. Renaming the repo should be its own PR in any case as we'll definitely want to check for other references to it.

**Pull request merged**